### ISSUE TITLE
Program interacts with OSK but not Maplestory 

### ISSUE THREAD

**Issue by qxh5696** [omgimanerd/python-maplestory-bot#2]
So I borrowed the code from Zero.py and changed it a bit to what I needed it to do for my Maplestory character and I start the program from my python command prompt and then select an active Maplestory window, but for some reason none of the commands executed on the OSK will be enacted on maple, only on the python command line that I executed the script on. Like you I am running windows 7. Is there something else that has to be done for the actions on the OSK to be run on Maple? 

TL;DR: When I have Maplestory open and I am actively using the OSK to interact with the game it works. When I run zero.py from my command prompt then select maplestory, the OSK is no longer controlled by the python script like I want it to be.

**Comment by omgimanerd** [comment]
After you run the Python script, select the Maplestory window so that actions from the OSK will be inputted into its context.
